data-gen/snoopAuditlog.py

```diff
@@ -117,12 +165,16 @@ def generate_coverage_report(openapi_spec, audit_log):
       # Populate tests[]
       if test_name not in tests.keys():
         tests[test_name] = []
+      if test_name not in test_sequences.keys():
+        test_sequences[test_name] = []
+      # if len(test_sequences[test_name]) < 30 and method_op in discovery_ops:
+      if method_op in discovery_ops:
+        # we should probably see if we can check if discovery is complete
+        continue # skip recording if we are using a discovery phase
```

Comment:
This is the main logic for 'skipping' discovery operations.
It might be interested to track if discovery is complete, then stop.
I'm open to ideas here.